### DISCUSSION TITLE
unsafe-eval

### DISCUSSION THREAD

**Discussion by Muhammad-Adam** [guasam/electron-react-app#42] [Q&A]
I have noticed that the `unsafe-eval` warning everytime.
![image](https://user-images.githubusercontent.com/10358628/194500803-0ecc9abc-c20f-40df-99b6-3382ee0088c1.png)

If i were to include the `meta` tag in the `index.html` file
`<meta http-equiv="Content-Security-Policy" content="script-src 'self';">`
The warning will disappear but there will be an error about refusing to evaluate string as javascript.

How do i go about removing the error?

**Comment by codesbiome** [comment]
@Muhammad-Adam 

The message you can see in Console is just a warning for development mode. Once you package the application for production it will not appear. 

If you wish to disable that warning, use this before creating `BrowserWindow` instance in following file: 

```js
// src/main/appWindow.ts

process.env['ELECTRON_DISABLE_SECURITY_WARNINGS'] = 'true'
```
